### PR TITLE
git-commit: add example

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/git-commit.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Commit only specific (already staged) files:
 
 `git commit {{path/to/file1}} {{path/to/file2}}`
+
+- Commit staged files to the repository with a message while allowing the commit be empty:
+
+`git commit -m "{{message}}" --allow-empty`
```
Comment (navarroaxel) [inline pages/common/git-commit.md:28]: Could you expand the arguments?
```suggestion
`git commit --message="{{message}}" --allow-empty`
```

Comment (CleanMachine1) [reply pages/common/git-commit.md:28]: I understand however its an incredible annoyance. Also most users would probably remember 

`m for message`

Comment (marchersimon) [reply pages/common/git-commit.md:28]: I wouldn't be too sure about that. Newcomers to git prabably don't have to commit that often that typing `--message` would be a problem.

Comment (marchersimon) [reply pages/common/git-commit.md:28]: At least I'd remove the `=`:
```suggestion
`git commit --message "{{message}}" --allow-empty`
```

File: pages/common/git-commit.md [hunk 2]
```diff
@@ -22,3 +22,7 @@
 - Commit only specific (already staged) files:
 
 `git commit {{path/to/file1}} {{path/to/file2}}`
+
+- Commit staged files to the repository with a message while allowing the commit be empty:
```
Comment (marchersimon) [inline pages/common/git-commit.md:26]: How about just
```suggestion
- Allow empty commits:
```

File: pages/common/git-commit.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/git-commit.md:28]: Hmm, I wonder if with such a common command as `git commit`, we should maybe keep the short options 🤔

I've never used `--message` before if I recall correctly 😅

Comment (marchersimon) [reply pages/common/git-commit.md:28]: It's the same situation as with #6083. Nobody would ever use `--sync --clean --clean`, but it's still good to know what they do, especially for beginners.

File: pages/common/git-commit.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/common/git-commit.md:26]: I don't think so. It doesn't describe it and doesn't fit with the page.

```suggestion
- Create a commit which has the ability to be empty with a message
```

Comment (bl-ue) [reply pages/common/git-commit.md:26]: That's too wordy.  How about
```suggestion
- Create an empty commit with no added, deleted, or changed files:
```

Comment (CleanMachine1) [reply pages/common/git-commit.md:26]: It isn't mandatory for it to empty, the flag just allows it to be.

Comment (CleanMachine1) [reply pages/common/git-commit.md:26]: If there are staged commits, it will put them in.

Comment (marchersimon) [reply pages/common/git-commit.md:26]: Another suggestion:
```suggestion
- Create a commit, even if there are no staged files:
```

File: pages/common/git-commit.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/git-commit.md:28]: yes but them commands are long and multiple flags, this is `m` if I was thinking for a message I would try -m.

The `S` for sync doesn't mean much to me, I'd immediately think search

File: pages/common/git-commit.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/common/git-commit.md:26]: Yes thats fine

File: pages/common/git-commit.md (hunk 1 above)
Comment (sbrl) [reply pages/common/git-commit.md:28]: I would suggest that this is a rare exception to the rule. Generally speaking you're going to be using this command a *lot*, so I'd suggest keeping the short version in this case, and maybe updating the description like we have on the `tar` page for example.

File: pages/common/git-commit.md (hunk 2 above)
Comment (sbrl) [reply pages/common/git-commit.md:26]: ....but surely you wouldn't use this unless you wanted an empty commit in the first place? Either of @marchersimon's suggestions would be fine.

Comment (CleanMachine1) [reply pages/common/git-commit.md:26]: You might use it in a script thats automated.

**Review by marchersimon — APPROVED** [review]
👍🏽

**Review by bl-ue — APPROVED** [review]
👍🏻

File: pages/common/git-commit.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/git-commit.md:28]: > Hmm, I wonder if with such a common command as `git commit`, we should maybe keep the short options thinking
> 
> I've never used `--message` before if I recall correctly sweat_smile

You haven't used message or just always used -m?

**Review by navarroaxel — APPROVED** [review]
